### DISCUSSION TITLE
PB as framework, first migration blank file

### DISCUSSION THREAD

**Discussion by mhabala** [pocketbase/pocketbase#2760] [Q&A]
Hello.

I have some tables stored and I would like to create a migration with them using PB as framework.
I have added Automigrate: isGoRun snippet from migration docs to my source.
I did do "go run main.go migrate create "migration" " and it created an empty file.

How do I fill the file up with database structure without records?
Was I supposed to do enable migrations before editing database structure and if so, can I do something retroactively to achieve desired result?

**Comment by ganigeorgiev** [comment]
[`migrate create`](https://pocketbase.io/docs/migrations/#migrate-create) creates a blank migration file.

[`migrate collections`](https://pocketbase.io/docs/migrations/#migrate-collections) can be used to generate a snapshot migration of your current collections configuration.

**Comment by mhabala** [comment]
When migrations are empty and I run PB and add new collections using interface, then close it and do migrate collections it creates migration with outdated collection.
